cypress/e2e/po/pages/explorer/cluster-project-members.po.ts

```diff
@@ -0,0 +1,40 @@
+import PagePo from '@/cypress/e2e/po/pages/page.po';
+import BaseResourceList from '@/cypress/e2e/po/lists/base-resource-list.po';
+import LabeledSelectPo from '@/cypress/e2e/po/components/labeled-select.po';
+import AsyncButtonPo from '@/cypress/e2e/po/components/async-button.po';
+
+export default class ClusterProjectMembersPo extends PagePo {
+  private static createPath(clusterId: string, tabId: string) {
+    return `/c/${ clusterId }/explorer/members#${ tabId }`;
+  }
+
+  static goTo(path: string): Cypress.Chainable<Cypress.AUTWindow> {
+    throw new Error('invalid');
+  }
+
+  constructor(clusterId: string = 'local', tabId: string = 'cluster-membership') {
+    super(ClusterProjectMembersPo.createPath(clusterId, tabId));
+  }
+
+  triggerAddClusterOrProjectMemberAction() {
+    return cy.get('.btn.role-primary.pull-right').click();
+  }
+
+  selectClusterOrProjectMember(name: string) {
+    const selectClusterOrProjectMember = new LabeledSelectPo(`[data-testid="cluster-member-select"]`, this.self());
```

Comment:
This is a bit niggly, but can you create a `select-principal.po` that `extends LabelSelectPo` and use it here? We're bound to do more things with this component and linking this to it will keep things neat.

Reply:
Sure thing.